yaksh/forms.py

```diff
@@ -274,6 +274,17 @@ def __init__(self, *args, **kwargs):
 class CourseForm(forms.ModelForm):
     """ course form for moderators """
 
+    def save(self, commit=True, *args, **kwargs):
```

Comment:
I personally feel that depending on s Character field for decision making is poor design in this case. These 2 fields allow greater flexibility because in case a design decision is made where Courses with codes are not supposed to be hidden from plain sight, then refactoring becomes a pain.
Moreover, this field will now allow us to hide special courses that students should not have access to like the Demo Yaksh Course.